### PR TITLE
LEP-239 Increase fixed employment allowance

### PR DESCRIPTION
**Pull request by ripan**
https://dsdmoj.atlassian.net/browse/LEP-239

---

## Checklists

Author: (before you ask people to review this PR)

- [x] Diff - review it, ensuring it contains only expected changes
- [x] Whitespace changes - avoid if possible, because they make diffs harder to read and conflicts more likely
- [ ] Changelog - add a line, if it meets the criteria
- [x] Secrets - no secrets should be added
- [x] Commit messages - say *why* the change was made
- [x] PR description - summarizes *what* changed and *why*, with a JIRA ticket ID
- [x] Tests pass - on CircleCI
- [x] Conflicts - resolve if Github reports them. e.g. with `git rebase main`

Reviewers remember:

- [ ] Jira ticket criteria are met
- [ ] Migrations - test migration and rollback: `rake db:migrate && rake db:rollback`

### CODE REVIEW

File: spec/requests/v6/assessments_controller/simple_spec.rb [hunk 1]
```diff
@@ -2137,6 +2137,48 @@ def cash_transactions(amount)
           end
         end
       end
+
+      context "MTR" do
+        let(:current_date) { Date.new(2525, 0o5, 0o1) }
```
Comment (starswan) [inline spec/requests/v6/assessments_controller/simple_spec.rb:2142]: octal dates?

File: features/employment/mtr/employment.feature [hunk 2]
```diff
@@ -0,0 +1,30 @@
+Feature:
+    "Employment"
+
+    Scenario: The client is employed, and receiving a benefit in kind
+        Given I am using version 6 of the API
+        And I create an assessment with the following details:
+            | client_reference_id | NP-FULL-1  |
+            | submission_date     | 2525-05-01 |
+        And I add the following applicant details for the current assessment:
+            | date_of_birth               | 1979-12-20 |
```
Comment (starswan) [inline features/employment/mtr/employment.feature:10]: could we use the shortened form here to reduce the test size?

File: features/employment/mtr/employment.feature [hunk 3]
```diff
@@ -0,0 +1,30 @@
+Feature:
+    "Employment"
+
+    Scenario: The client is employed, and receiving a benefit in kind
```
Comment (starswan) [inline features/employment/mtr/employment.feature:4]: the scenario (and test) seem to say more than I would expect as we're only showing MTR employment threshold

File: features/employment/mtr/employment.feature [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            | ccms_code | client_involvement_type |
+            | SE013     | A                       |
+      And I add the following employment details:
+        | client_id |     date     |  gross | benefits_in_kind  | tax    | national_insurance  |
+        |     C     |  2022-06-22  | 500.00 |      100          | -55.00 |       -25.0         |
+        |     C     |  2022-07-22  | 500.00 |      100          | -55.00 |       -25.0         |
+        |     C     |  2022-08-22  | 500.00 |      100          | -55.00 |       -25.0         |
+        When I retrieve the final assessment
+        Then I should see the following "employment" details:
+            | attribute                  | value  |
```
Comment (starswan) [inline features/employment/mtr/employment.feature:24]: all we care about here is the deduction IMHO?

File: spec/requests/v6/assessments_controller/simple_spec.rb [hunk 5]
```diff
@@ -2137,6 +2137,48 @@ def cash_transactions(amount)
...
+
+        context "when threshold 'fixed_employment_allowance' is set to 60" do
```
Comment (starswan) [inline spec/requests/v6/assessments_controller/simple_spec.rb:2144]: this isn't even true (its 66) but the context should describe the 'given' which is MTR?

File: spec/requests/v6/assessments_controller/simple_spec.rb (hunk 1 above)
Comment (ripan) [reply spec/requests/v6/assessments_controller/simple_spec.rb]: Removed duplicate test

File: spec/requests/v6/assessments_controller/simple_spec.rb (hunk 5 above)
Comment (ripan) [reply spec/requests/v6/assessments_controller/simple_spec.rb]: Removed request spec and added unit test instead

**Review by starswan — APPROVED** [review]

**Pull request merged**